src/lib/ngsi/ContextAttributeVector.cpp

```diff
@@ -311,13 +311,19 @@ ContextAttribute* ContextAttributeVector::get(unsigned int ix)
   return NULL;
 }
 
+const ContextAttribute* ContextAttributeVector::get(unsigned int ix) const
+{
+  if (ix < vec.size())
+    return vec[ix];
+  return NULL;
+}
```

Comment:
Instead of adding get() methods, I would remove them, and use only the operator[] instead.
The get() methods are old, we started to use operator[] instead a long time ago.

Perhaps, I've missed something about this ...